### DISCUSSION TITLE
Use DatePicker Calendar without Input

### DISCUSSION THREAD

**Discussion by ngenereux-ps** [vmware/clarity#6016] [Q&A] [answered]
I'd like to use the calendar component of the datepicker to allow users to select a date. When the date changes, associated data in another part of the view will change. I want the user to be able to quickly search through different days, so I don't want the picker to disappear each time they change the date.

Is there a way to use the clr-daypicker component without the input component? I've been poking through the code and I don't see an obvious solution. This is an Angular project.

**Comment by Jinnie** [comment, marked as answer]
No, you can't do it with the Clarity Angular datepicker.
Such level of flexibility will probably be possible in the Date picker implementation in Clarity Core.
It's still too early for a certain answer, but this discussion will help us in the component architecture planning. Thanks

**Reply by ngenereux-ps** [reply]
Thanks for the answer! Does the Clarity Core implementation of the date picker have an associated issue or timeline so I can track it?
